src/pdf/ShoppingList/ShoppingListPDF.tsx

```diff
@@ -0,0 +1,235 @@
+import {
+    BlockProps,
+    ClientSummary,
+    Item,
+    ShoppingListPDFProps,
+} from "@/pdf/ShoppingList/dataPreparation";
+
+const styles = StyleSheet.create({
+    paper: {
+        margin: "0.75in",
+        lineHeight: 1.5,
+    },
+    cell: {
+        display: "flex",
+        padding: "20 5 5",
+    },
+    pdfHeader: {
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "space-between",
+    },
+    logoStyling: {
+        display: "flex",
+        maxWidth: "20%",
+        alignSelf: "center",
+    },
+    infoBlock: {
+        display: "flex",
+        flexDirection: "row",
+        borderStyle: "solid",
+        border: 1,
+    },
+    infoCell: {
+        display: "flex",
+        width: "50%",
+        padding: 5,
+    },
+    table: {
+        display: "flex",
+        flexDirection: "column",
+        borderStyle: "solid",
+        border: 1,
+        padding: 5,
+    },
+    tableRow: {
+        display: "flex",
+        flexDirection: "row",
+        borderBottomStyle: "solid",
+        borderBottom: 1,
+        paddingVertical: 4,
+        textAlign: "center",
+    },
+    tableItemDescription: {
+        width: "25%",
+    },
+    tableQuantity: {
+        width: "35%",
+    },
+    tableNotes: {
+        width: "30%",
+    },
+    tableDone: {
+        width: "10%",
+    },
+    title: {
+        fontSize: "30pt",
+        fontFamily: "Helvetica-Bold",
+    },
+    subtitle: {
+        fontSize: "24pt",
+        fontFamily: "Helvetica",
+    },
+    keyText: {
+        fontSize: "12pt",
+        fontFamily: "Helvetica-Bold",
+        textTransform: "capitalize",
+    },
+    normalText: {
+        fontSize: "12pt",
+        fontFamily: "Helvetica",
+    },
+    input: {
+        fontSize: "12pt",
+        fontFamily: "Helvetica-Bold",
+        paddingTop: 20,
+    },
+});
+
+const OneLine: React.FC<{ header: string; value: string }> = ({ header, value }) => {
+    return (
+        <Text style={styles.keyText}>
+            {header}: <Text style={styles.normalText}>{value}</Text>
+        </Text>
+    );
+};
+
+const TableHeadings: React.FC<{}> = () => {
+    return (
+        <View style={styles.tableRow}>
+            <View style={styles.tableItemDescription}>
+                <Text style={styles.keyText}>Item Description</Text>
+            </View>
+            <View style={styles.tableQuantity}>
+                <Text style={styles.keyText}>Quantity</Text>
+            </View>
+            <View style={styles.tableNotes}>
+                <Text style={styles.keyText}>Notes</Text>
+            </View>
+            <View style={styles.tableDone}>
+                <Text style={styles.keyText}>Done</Text>
+            </View>
+        </View>
+    );
+};
+
+const ItemToRow: React.FC<Item> = (item) => {
+    return (
+        <View style={styles.tableRow}>
+            <View style={styles.tableItemDescription}>
+                <Text style={styles.normalText}>{item.description}</Text>
```

Comment:
As mentioned above, some of them could be done like this: [styles.whatever, { property: value}] for readability